### PR TITLE
Fix some typos of Japanese situation repost 0320

### PR DESCRIPTION
**Pull request by satotake**
I have read it and proposed fixation of a few of typos here

Thanks for your hard work @tomoyuki-nezu , @fzhang12 and @cassiawag .

### CODE REVIEW

**Comment by fzhang12** [comment]
Thanks for improving.

**Comment by cassiawag** [comment]
Thanks for the edits @satotake! Can I get a confirmation that you've reviewed this @fzhang12? I'll add these then.

File: narratives/ncov_sit-rep_ja_2020-03-20.md [hunk 1]
```diff
@@ -255,10 +255,10 @@ COVID-19 が突然爆発したように見えるかもしれません。
 <br>
 - 多くの地域で地域伝搬の証拠が見られます。これを超えて、過去の伝搬により、今後数週間で明らかになる多くの地域流行がすでに進行中であることが予想されます。  
 <br>
-- 社会的距離をとおして地域の発生を制御することは:  
-  - #FlattenTheCurve （流行曲線の抑制）を使用して、医療システムに対する感染者の集中を防ぎます。  
+- 社会的距離をとおして地域の感染流行を制御することは:  
+  - #FlattenTheCurve （流行曲線の抑制して）、医療システムに対する感染者の集中を防ぎます。
```
Comment (fzhang12) [inline narratives/ncov_sit-rep_ja_2020-03-20.md]: for the following line, I prefer the previous one. Except this point, others are pretty fine and actually adding fluency.

> - #FlattenTheCurve （流行曲線の抑制）を使用して

**Comment by fzhang12** [comment]
@cassiawag I just finished my comment. Only a small point I don't agree with, other changes are good for my taste.

**Comment by cassiawag** [comment]
Thanks @fzhang12! @satotake, once you address, I will merge it. Thank you so much for your help! Please add your name and website under translators and translatorLinks too.

File: narratives/ncov_sit-rep_ja_2020-03-20.md (hunk 1 above)
Comment (satotake) [reply narratives/ncov_sit-rep_ja_2020-03-20.md]: rollbacked. Thanks for rapid reviewing!

**Comment by satotake** [comment]
@cassiawag 
I have reflected the comment of @fzhang12 and added name.

**Comment by cassiawag** [comment]
Edits are live! Thank you to you both.

**Pull request merged**